pwndbg/disasm/__init__.py

```diff
@@ -100,11 +100,13 @@ def get_one_instruction(address):
         pwndbg.disasm.arch.DisassemblyAssistant.enhance(ins)
         return ins
 
-def one(address=None):
+
+
+def one(address='unset_argument'):
+    if address == 'unset_argument':
```

Comment:
well you want to distinguish between non set arugment and non argument. the whole codebase is build like it passes None and that can't be set to PC. however this whole function checks for none dont make any sense at all if ir returns None when being None :P 
so a) we fix _all_ million of calls to this func to never pass None
b) we dont allow to call it without argument
c) we have a special value to distiguis between none arguments and calls without argument